src/main/resources/base-target.yaml

```diff
@@ -51,6 +51,8 @@ target:
         merging:
           # If XML merging should be done when indexing XML
           enabled: true
+          inherits:
+            field: inheritsFrom_smv
```

Comment:
I think I rather have a `inheritsFrom` without `_smv`, and defined in the ES settings, just like `includedDescriptors`. I like having Crafter specific fields without a postfix. Unless doing this avoids having to do a re-index? But don't we need a re-index anyway? @sumerjabri let me know what you think too.

Reply:
yeah, I did it that way to avoid changing both ES & Solr schemas, that needs not only a reindex but creating a new index

Reply:
Ok, re-creating an index is not worth it, let's go with the postfix.